numpy/core/src/umath/ufunc_object.c

```diff
@@ -2013,7 +2030,8 @@ PyUFunc_GeneralizedFunction(PyUFuncObject *ufunc,
     /* The sizes of the core dimensions (# entries is ufunc->core_num_dim_ix) */
     npy_intp *core_dim_sizes = inner_dimensions + 1;
     int core_dim_ixs_size;
-
+    /* swapping around of axis (TODO: allocate this when needed) */
+    int remap_axis[NPY_MAXARGS][NPY_MAXDIMS];
```

Comment:
not sure whether it is work allocating: 32*32 elements; same holds for `op_axes_arrays` above.